src/test/suite/index.ts

```diff
@@ -29,52 +25,54 @@ export function run(): Promise<void> {
   const testsRoot = path.join(__dirname, '..');
 
   return new Promise((c, e) => {
-    glob('**/**.test.js', {
-      cwd: testsRoot,
-      ignore: ['**/webview-app/**/*.js']
-    }, (err, files) => {
-      if (err) {
-        return e(err);
-      }
+    glob(
+      '**/**.test.js',
+      {
+        cwd: testsRoot,
+        ignore: ['**/webview-app/**/*.js']
+      },
+      (err, files) => {
+        if (err) {
+          return e(err);
+        }
 
-      // Activate the extension.
-      mdbTestExtension.testExtensionContext = new TestExtensionContext();
-      mdbTestExtension.testExtensionController = new MDBExtensionController(
-        mdbTestExtension.testExtensionContext
-      );
-      mdbTestExtension.testExtensionController.activate();
+        // Activate the extension.
+        mdbTestExtension.testExtensionContext = new TestExtensionContext();
+        mdbTestExtension.testExtensionController = new MDBExtensionController(
+          mdbTestExtension.testExtensionContext,
+          false
```

Comment:
We set a `neetTelemery` flat to false, which means all telemetry events will be disabled in test mode.